android/expoview/src/main/java/host/exp/exponent/kernel/Kernel.java

```diff
@@ -550,9 +552,12 @@ public void handleIntent(Activity activity, Intent intent) {
       }
     }
 
-    if (uri != null) {
+    if (uri != null && shouldOpenUrl(uri)) {
       if (Constants.INITIAL_URL == null) {
-        // We got an "exp://" link
+        // We got an Expo Go app link (could be exp://, exps://, http://, or https://)
+        if (!uri.getScheme().equals("exp")) {
```

Comment:
Shouldn't we change http -> exp and https -> exps?

Reply:
actually we can remove this now that we're not supporting project pages i.e. "https://expo.io/@username/slug" as universal links anymore